extension/index.js

```diff
@@ -96,6 +98,28 @@ async function openDropdown({currentTarget: indicator}) {
 		const container = select('.NPG-dropdown', dropdown);
 		empty(container);
 		container.append(...list);
+
+		const forms = select.all(('.NPG-dropdown form'));
+		for (const form of forms) {
+			delegate(form, 'button', 'click', async event => {
```

Comment:
You don't need a `for` loop here. Use the snippet as suggested, without passing the `form` argument.